components/Dashboard.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <div>
```

Comment:
What if I add more markup besides header?

Makes send about header being on the `<template>` for `<index.vue>`